src/index.html

```diff
@@ -34,158 +35,219 @@
         titel="Sign up"
         class="wp-block-button__link has-text-align-center wp-element-button"
         href="https://app.checkrideai.com/purchase"
-        >Get now</a
+        >Start preparing</a
       >
     </div>
 
     <div class="wp-block-button is-style-outline">
       <a
         class="wp-block-button__link wp-element-button"
-        href="#faq"
+        href="#why-checkride-ai"
         title="Learn more"
         >Learn more</a
       >
     </div>
   </div>
 
-  <div
-    class="wp-block-group alignwide has-global-padding is-layout-constrained wp-container-14 wp-block-group-is-layout-constrained"
-  >
-    <div
-      class="wp-block-columns alignwide is-layout-flex wp-container-13 wp-block-columns-is-layout-flex"
-    >
-      <div
-        class="wp-block-column is-layout-flow wp-block-column-is-layout-flow"
-        style="flex-basis: 10%"
-      ></div>
+  <div id="video" style="
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 50px;
+    padding-bottom: 50px;
+  ">
+    <video style="width: 100%; max-width: 70%; height: auto;" autoplay muted>
```

Comment:
```suggestion
    <video style="width: 100%; max-width: 70%; height: auto;" autoplay muted loop>
```
makes it autoloop

Reply:
also, it's relatively unreadable on mobile:
![image](https://github.com/EpicPi/checkrideai-web/assets/9906905/b5a508c7-1739-4e48-be36-847fff0f78b0)
getting rid of the max-width prop helps, but real solution is probably to show a different video if on mobile - one that was recorded on a narrower screen.